.github/workflows/ci-build.yaml

```diff
@@ -103,15 +103,19 @@ jobs:
         with:
           python-version: '3.x'
           cache: pip
-      - name: Install and start K3S
+      - uses: AbsaOSS/k3d-action@v2
```

Comment:
Is there anywhere to explicitly specify the k3s version?

Reply:
Good spot. I switched from manual install. 

https://github.com/AbsaOSS/k3d-action

I'm now not clear what K3s we're running. I need to re-visit this.

Reply:
Hmmm. `docker.io/rancher/k3s:v1.22.7-k3s1`